bin/anycablebility

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env ruby
+
+require 'bundler/setup'
+require 'docopt'
```

Comment:
- I don't know any project that uses it

- A lot of popular Ruby projects use OptionParser (Sidekiq, Rubocop to name a few)

- It has not been maintained for a rather long time (https://ossert.evilmartians.io/docopt)